server/sonar-web/src/main/js/apps/system/components/info-items/HealthItem.tsx

```diff
@@ -19,31 +19,24 @@
  */
 import * as React from 'react';
 import * as classNames from 'classnames';
-import { HealthType, HealthCause } from '../../types';
+import HealthCauseItem from './HealthCauseItem';
+import { HealthType, HealthCause } from '../../../../api/system';
 
 interface Props {
-  bigger?: boolean;
   className?: string;
   health: HealthType;
-  healthCauses: HealthCause[];
+  healthCauses?: HealthCause[];
 }
 
-export default function HealthItem({ bigger, className, health, healthCauses }: Props) {
-  const hasCauses = health !== HealthType.GREEN && healthCauses.length > 0;
+export default function HealthItem({ className, health, healthCauses }: Props) {
+  const hasHealthCauses = healthCauses && healthCauses.length > 0 && health !== HealthType.GREEN;
   return (
     <div className={classNames('system-info-health-info', className)}>
-      {hasCauses &&
-        healthCauses.map((cause, idx) => (
-          <span
-            key={idx}
-            className={classNames(
-              'spacer-right alert',
-              health === HealthType.RED ? 'alert-danger' : 'alert-warning'
-            )}>
-            {cause.message}
-          </span>
+      {hasHealthCauses &&
+        (healthCauses as HealthCause[]).map((cause, idx) => (
```

Comment:
instead of type-cast, use can use `healthCauses!.map(...)`

Reply:
:+1: I didn't know there was something like that! Thanks!